builtin/game/chatcommands.lua

```diff
@@ -479,7 +497,11 @@ end
 core.register_chatcommand("emergeblocks", {
 	params = "(here [<radius>]) | (<pos1> <pos2>)",
 	description = "Load (or, if nonexistent, generate) map blocks "
-		.. "contained in area pos1 to pos2 (<pos1> and <pos2> must be in parentheses)",
+		.. "contained in area from "
+		.. core.colorize(core.COLOR_PARAM, "<pos1>").." to "
+		.. core.colorize(core.COLOR_PARAM, "<pos2>").." "
+		.. "(" .. core.colorize(core.COLOR_PARAM, "<pos1>").. " and "
+		.. core.colorize(core.COLOR_PARAM, "<pos2>").." must be in parentheses)",
```

Comment:
A local helper function to search and colourize `<%s>` might improve the readability for these code parts.

Reply:
l agree, using string.format you can do the same as in printf (see $ man 3 printf), e.g. 
```lua
("f(%d) = %d"):format(x, myfunc(x))
``` 
instead of 
```lua
"f(" .. x .. ") = " .. myfunc(x)
```
In some situations this makes code more readable.

Reply:
Or when keeping the old description:
```Lua
("contained in area <pos1> to <pos2> (<pos1> and <pos2> must be in parentheses)")
:gsub("(%<%S+%>)", core.colorize(core.COLOR_PARAM, "%1"))
```